test/unit/utils.test.js

```diff
@@ -112,15 +112,8 @@ describe('getDefaultFacets', () => {
     const defaultFacets = getDefaultFacets();
 
     expect(defaultFacets).to.eql({
-      owner: [],
-      contributorLiteral: [],
       materialType: [],
-      issuance: [],
-      publisher: [],
       language: [],
-      mediaType: [],
-      subjectLiteral: [],
-      creatorLiteral: [],
       dateAfter: {},
```

Comment:
The format for `dateAfter` and `dateBefore` will probably be number?

Reply:
Not sure. Something we have to figure out once you start doing more of the date filter. Also, in your PR you have it as a string.

Reply:
Yeah but if we are going to change the type based on your comment it will be number.

Reply:
Sure, again, that's something you need to settle down in your PR since you're working on that filter. You have to decide what makes more sense for the Acceptance Criteria and how the app works. The existing `dateAfter: {}` is just what was there from before.

Reply:
Updated on my PR. They will be text.

Reply:
Do you mean string?

Reply:
yeah string! I mean input type will be text.